README.md

```diff
@@ -151,19 +151,9 @@ correct permissions.  Try then:
 
     $ sudo dfu-util -a 0 -d 0483:df11 -D build-PYBV10/firmware.dfu
 
-Building the documentation locally
-----------------------------------
+Documentation
```

Comment:
Do you think we need this small section at all?  If people want to compile the docs they'll probably anyway be looking in the docs dir and see the README.md there.